requirements.txt

```diff
@@ -1,9 +1,9 @@
-pybtex==0.20.1
-Pygments>=2.0,<3
-Sphinx==1.3.6
-sphinx_rtd_theme==0.1.9
-sphinxcontrib-bibtex==0.3.3
-numpy==1.11.0
-matplotlib==1.5.1
+pybtex
```

Comment:
I don't think pip has any sort of system for determining dependencies like python 3.